### DISCUSSION TITLE
Bosquejo de página de contacto para miembros

### DISCUSSION THREAD

**Discussion by Brian-Almada** [rpgrca/proyecto-ispc-fullstack#90] [Ideas]
![diseñohtml-contactomiembro](https://user-images.githubusercontent.com/100881466/193612523-d12b53d4-0f81-4f8e-b5d1-69c79dfdbf63.png)

Ese sería un diseño de formulario con la opción de mandar un mail también.
Pero, para que solo sea formulario se puede sacar la opción mail y quedaría así.

![diseñohtml-contactmiembro](https://user-images.githubusercontent.com/100881466/193613947-5ea795cf-69ef-4b20-8214-8e06011dfebb.png)

Cuál de los dos sería una mejor opción?

**Comment by NataliaAlvarezIspc** [comment]
Hola! para mi la que va es la segunda opción. La primera abriria una aplicación que no siempre es la que un@ utiliza para el mail.
